src/custom/api/gnosisProtocol/api.ts

```diff
@@ -275,52 +274,89 @@ const UNHANDLED_ORDER_ERROR: ApiErrorObject = {
   description: ApiErrorCodeDetails.UNHANDLED_CREATE_ERROR,
 }
 
+function _handleError<P extends Context>(error: any, response: Response, params: P, operation: 'ORDER' | 'QUOTE') {
+  // Create a new sentry error OR
+  // use the previously created and rethrown error from the try block
+  const sentryError =
+    error?.sentryError ||
+    constructSentryError(error, response, {
+      message: `Potential backend error detected - status code: ${response.status}`,
```

Comment:
nit: Maybe simpler is `CowAPI error`, not even the status is required if that is added as part of the tags. The simpler the message, the better.